production/sample/grafana-cloud/main.jsonnet

```diff
@@ -0,0 +1,148 @@
+local ksm = import 'github.com/grafana/jsonnet-libs/kube-state-metrics/main.libsonnet';
+local node_exporter = import 'github.com/grafana/jsonnet-libs/node-exporter/main.libsonnet';
+local gragent = import 'grafana-agent/v2/main.libsonnet';
+local k = import 'ksonnet-util/kausal.libsonnet';
+local tk = import 'tk';
+local tns_mixin = import 'tns-mixin/mixin.libsonnet';
+
+{
+  _config+:: {
+    apiKey: error '$._config.apiKey must be defined',
```

Comment:
can just one apikey write to all the different endpoints?

Reply:
yep, a `MetricsPublisher` api key can write to all backends